.github/workflows/ci_pipeline.yml

```diff
@@ -230,7 +303,7 @@ jobs:
       - name: Install Conda environment with Micromamba
         uses: mamba-org/setup-micromamba@v1
         with:
-          environment-file: "etc/environment-base.yml"
+          environment-file: "etc/environment-tests.yml"
```

Comment:
Are we sure this is enough? Can you try running this job locally and see if the website looks good?

Reply:
Now the environment for documentation has all needed dependencies